lib/ansible/modules/system/aix_filesystem.py

```diff
@@ -0,0 +1,600 @@
+        Gigabytes.
+        If no M or G the value will be 512-byte blocks.
+        If "+" is specified in begin of value, the value will be added. If "-"
+        is specified in begin of value, the value will be removed. If "+" or
+        "-" is not specified, the total value will be the specified.
+        Size will respects the LVM AIX standards.
+    required: false
+  state:
+    description:
+      - Controls the file system state.
+        C(present) check if file system exists, creates or resize.
+        C(absent) removes existing file system if already C(unmounted).
+        C(mounted) checks if the file system is mounted or mount the file
+        system.
+        C(unmounted) check if the file system is unmounted or unmount the file
+        system.
+    choices: [present, absent, mounted, unmounted]
+    default: present
+    required: true
+  vg:
+    description:
+      - Specifies an existing volume group (VG).
+    required: true
+notes:
+  - For more C(attributes), please check "crfs" AIX manual.
+'''
+
+EXAMPLES = '''
+- name: Create filesystem in a previously defined logical volume.
+  aix_filesystem:
+    device: testlv
+    filesystem: /testfs
+    state: present
+- name: Creating NFS filesystem from nfshost.
+  aix_filesystem:
+    device: /home/ftp
+    nfs_server: nfshost
+    filesystem: /home/ftp
+    state: present
+- name: Creating a new file system without a previously logical volume.
+  aix_filesystem:
+    filesystem: /newfs
+    size: 1G
+    state: present
+    vg: datavg
+- name: Unmounting /testfs.
+  aix_filesystem:
+    filesystem: /testfs
+    state: unmounted
+- name: Resizing /mksysb to +512M.
+  aix_filesystem:
+    filesystem: /mksysb
+    size: +512M
+    state: present
+- name: Resizing /mksysb to 11G.
+  aix_filesystem:
+    filesystem: /mksysb
+    size: 11G
+    state: present
+- name: Resizing /mksysb to -2G.
+  aix_filesystem:
+    filesystem: /mksysb
+    size: -2G
+    state: present
+- name: Remove NFS filesystem /home/ftp.
+  aix_filesystem:
+    filesystem: /home/ftp
+    rm_mount_point: yes
+    state: absent
+- name: Remove /newfs.
+  aix_filesystem:
+    filesystem: /newfs
+    rm_mount_point: yes
+    state: absent
+'''
+
+RETURN = '''
+changed:
+  description: Return changed for aix_filesystems actions as true or false.
+  returned: always
+  type: boolean
+  version_added: "2.5"
+msg:
+  description: Return message regarding the action.
+  returned: always
+  type: string
+  version_added: "2.5"
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ismount import ismount
+import re
+
+
+def _fs_exists(module, filesystem):
+    """
+    Check if file system already exists on /etc/filesystems.
+
+    :param module: Ansible module.
+    :param filesystem: filesystem name.
+    :return: True or False.
+    """
+    lsfs_cmd = module.get_bin_path('lsfs', True)
+    rc, lsfs_out, err = module.run_command(
+        "%s -l %s" % (lsfs_cmd, filesystem))
```

Comment:
Please put this on a single line. Lines can be 159 characters wide.

Reply:
I thought it was optional, anyway. Indeed. :)

Reply:
Right but this seems arbitrary, cut off at 40 characters ?
For readability I think it's better to not split lines if it is not required.